packages/block-library/src/post-comments/edit.js

```diff
@@ -174,21 +187,27 @@ export default function PostCommentsEdit( {
 												'To get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.'
 											) }
 											<br />
-											{ __(
-												'Commenter avatars come from'
-											) }{ ' ' }
-											<a href="https://gravatar.com/">
-												Gravatar
-											</a>
-											.
+											{ createInterpolateElement(
+												__(
+													'Commenter avatars come from <a>Gravatar</a>'
+												),
+												{
+													a: (
+														/* eslint-disable-next-line jsx-a11y/anchor-has-content */
+														<a href="https://gravatar.com/" />
+													),
+												}
+											) }
 										</p>
 									</div>
 
 									<div className="reply">
 										<a
 											className="comment-reply-link"
 											href="#top"
-											aria-label="Reply to A WordPress Commenter"
+											aria-label={ __(
+												'Reply to A WordPress Commenter'
```

Comment:
Could this use the following to avoid new strings?

```js
/* translators: Comment reply button text. %s: Comment author name. */
sprintf( 
  __( 'Reply to %s' ),
  __( 'A WordPress Commenter' )
);
```